config/webpack.prod.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path');
+const TerserPlugin = require('terser-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+module.exports = {
+  mode: 'production',
+  entry: './src/index.js',
+  output: {
+    filename: '[name].bundle.js',
+    path: path.join(__dirname, '../dist/prod')
+  },
+  resolve: {
+    extensions: [
+      '.js',
+      '.json'
+    ]
+  },
+  plugins: [
+    new CleanWebpackPlugin(),
+    new HtmlWebpackPlugin({
+      title: 'production mode',
+      template: 'src/index.html',
+      filename: 'index.html'
+    }),
+    new MiniCssExtractPlugin({
+      filename: '[name].css',
+    })
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.m?js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: [
+              '@babel/preset-env'
+            ]
+          }
+        }
+      },
```

Comment:
I suggest you creating third `webpack.common.js` confit and keep all the common configurations for dev and prod in it. Please see https://webpack.js.org/guides/production/#setup